src/bsoncxx/CMakeLists.txt

```diff
@@ -135,6 +135,15 @@ set(bsoncxx_target_list "")
 if(BSONCXX_BUILD_SHARED)
     bsoncxx_add_library(bsoncxx_shared "${BSONCXX_OUTPUT_BASENAME}" SHARED)
     list(APPEND bsoncxx_target_list bsoncxx_shared)
+
+    if(WIN32)
```

Comment:
Confirming: should this be `if(WIN32)` (target system is Windows) or `if(MSVC)` (compiling with MSVC/Visual C++)?

Reply:
I expect "yes", since non-MSVC compilers may be able to build the `.rc` file. Though the C++ driver does not currently test with MinGW, MinGW can build the `.rc` file using `windres`.